tests/sentry/auth/test_superuser.py

```diff
@@ -71,10 +72,19 @@ def build_request(
     def test_ips(self):
         user = User(is_superuser=True)
         request = self.make_request(user=user)
+        request._body = json.dumps(
+            {
+                "superuserAccessCategory": "Edit organization settings",
+                "superuserReason": "Edit organization settings",
+            }
+        )
         request.META["REMOTE_ADDR"] = "10.0.0.1"
 
         # no ips = any host
         superuser = Superuser(request, allowed_ips=())
+        request.session[SsoSession.django_session_key(superuser.org_id)] = {
+            "ts": timezone.now().timestamp()
```

Comment:
Is this just a placeholder?

If there's any chance of the timestamp affecting test behavior, I'd rather replace it with a constant or something derived from `self.current_datetime`.